File: csharp/Dedicnost/Dedicnost/Urednik.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Dedicnost
{
    internal class Urednik: Clovek
    {
        public int Uroven { get; set; }

        public Urednik(string jmeno, int vek, int uroven) : base(jmeno, vek) {
        
            Uroven = uroven;
        }

        public override void Pozdrav()
        {
            Console.WriteLine($"Ahoj posila {Jmeno} ve veku {Vek} je dospely {jeDospely} ze tridy Urednik urovne {Uroven}");
        }
    }
}

File: java/Dedicnost/src/main/java/org/example/Urednik.java
package org.example;

public class Urednik extends Clovek{

    public int uroven;

    public Urednik(String jmeno, int vek, int uroven) {

        super(jmeno, vek);
        this.uroven = uroven;
    }

    @Override
    public void Pozdrav()
    {
        System.out.println("Ahoj posila " + jmeno + " ve veku " + vek + " je dospely " + jeDospely + " ze tridy Urednik urovne " + uroven);
    }
}
